### PR TITLE
feat: embedded DB (spawning server from client)

### PR DESCRIPTION
**Pull request by dandv**
Documenting the "embedded DB" option for Python and JavaScript on Linux.

**[Staging page](https://6423dd5319cb665c246ec2f2--tangerine-buttercream-20c32f.netlify.app/developers/weaviate/installation/embedded)**

### CODE REVIEW

File: developers/weaviate/installation/embedded.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+:::warning Experimental phase
+Embedded Weaviate is still in the **Experimental** phase.
+
+Some of the APIs and parameters might change over time, as we work towards a perfect implementation.
+:::
+
+### How is this possible?
+
+With every Weaviate [release](https://github.com/weaviate/weaviate/releases) we also publish a single-executalbe Linux binary ([see assets](https://github.com/weaviate/weaviate/releases)).
```
Comment (dirkkul) [inline developers/weaviate/installation/embedded.md:19]: typo executable

Comment (dirkkul) [reply developers/weaviate/installation/embedded.md:19]: > publish a single-executalbe

what is single supposed to mean here? We also publish ARM64-linux

File: developers/weaviate/installation/embedded.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## Embedded options
+
+The Weaviate server spwaned from the client can be configured via parameters passed to the at instantiation time, and via environment variables. All parameters are optional.
+
+| Parameter | Type | Description | Default value | Environment variable |
+| --------- | ---- | ----------- | ------------- | ------------------- |
+| persistence_data_path | string | Directory where the files making up the database are stored | `/.local/share/weaviate` | `XDG_DATA_HOME` |
+| binary_path | string | Where the binary is downloaded. If deleted, the client will download the binary again. | `~/.cache/weaviate-embedded` | `XDG_CACHE_HOME` |
+| version | string | Full URL pointing to the Linux AMD64 binary | [Varies](https://github.com/weaviate/weaviate/releases) | |
```
Comment (dirkkul) [inline developers/weaviate/installation/embedded.md:35]: or ARM64

File: developers/weaviate/installation/embedded.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+| port | integer | Which port the Weaviate server will listen to. Useful when running multiple instances in parallel. | 6666 | |
+| hostname | string | Hostname/IP to bind to. | 127.0.0.1 | |
+| cluster_hostname | string | The label for the cluster hostname | "embedded" | CLUSTER_HOSTNAME |
+| additional_env_vars | key = value | Useful to pass additional environment variables to the server, such as API keys. | |
```
Comment (dirkkul) [inline developers/weaviate/installation/embedded.md:39]: I'd replace `key = value` with `key: value`

File: developers/weaviate/installation/embedded.md (hunk 1 above)
Comment (sebawita) [reply developers/weaviate/installation/embedded.md:19]: good point
We can change it to:
we also publish executable Linux binaries

would that work?

File: developers/weaviate/installation/embedded.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+* [Python client](../client-libraries/python.md) – `v3.15.4` or newer
+* [TypeScript client](https://github.com/weaviate/typescript-client) – `v1` or newer
+    * Note: the TypeScript client is still in Beta.
+
+## More Resources
+
+import DocsMoreResources from '/_includes/more-resources-docs.md';
+
+<DocsMoreResources />
```
Comment (dirkkul) [inline developers/weaviate/installation/embedded.md:71]: I would add that this is especially useful for notbook environments (eg jupyter). If you have a normal python script, weaviate will shut down before when the script is done. I don't think many people would expect that

**Pull request merged**